app/controllers/camaleon_cms/admin/media_controller.rb

```diff
@@ -93,6 +84,7 @@ def actions
 
       def local_url?(url)
```

Comment:
@texpert We don't need this method anymore, do we? It's no longer called anywhere as far as I can tell.

Reply:
Thanks, @brian-kephart !

Fixed in https://github.com/owen2345/camaleon-cms/pull/1048/commits/974425acd27f17fb8dd6c6f07a735b4402fea064